frontend/src/components/Admin/CoffeeChats/CoffeeChats.tsx

```diff
@@ -58,6 +59,11 @@ const CoffeeChats: React.FC = () => {
           ))}
         </div>
       </div>
+      <div className={styles.buttonContainer}>
+        <Button>
+          <Link href="/admin/coffee-chats/dashboard">View Coffee Chats Dashboard</Link>
```

Comment:
Could you wrap the `Button` with the `Link`? Or add an `onClick` function to the `Button`.

I'm getting this weird behavior where I click on the button, but it doesn't always navigate to the page right away unless I click on the text directly.